Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.cpp

```diff
@@ -623,15 +623,24 @@ namespace AzFramework
     {
         Camera nextCamera = targetCamera;
 
-        const auto pivotDirection = targetCamera.m_offset.GetNormalized();
-        nextCamera.m_offset -= pivotDirection * delta;
-        const auto pivotDot = targetCamera.m_offset.Dot(nextCamera.m_offset);
-        const auto distance = nextCamera.m_offset.GetLength() * AZ::GetSign(pivotDot);
+        // handle case where pivot and offset may be the same to begin with
```

Comment:
nit: first letter upper case

Reply:
I usually follow the convention (through this file and others) that inline comments in .cpp files always are lowercase (API comments start with an uppercase letter always) - I find this easier to follow and is the convention followed throughout this file so I'll leave for now if that's cool